docs/source/api/soil.md

```diff
@@ -14,15 +14,24 @@ kernelspec:
   name: vr_python3
 ---
 
-# API reference for `soil` modules
+# API reference for `models.soil` modules
 
 This page contains the detailed documentation of the functions and classes in the
 `virtual_rainforest.soil` modules.
 
-## The `virtual_rainforest.soil.model` module
+## The `virtual_rainforest.models.soil.model` module
 
 ```{eval-rst}
-.. automodule:: virtual_rainforest.soil.model
+.. automodule:: virtual_rainforest.models.soil.model
+    :autosummary:
+    :members:
+    :special-members: __init__
+```
```

Comment:
Depending on how large these are likely to be, could use subpages within a folder, so:
```
soil_overview.md
soil/model.md
soil/carbon.md
```

Reply:
Yeah that is probably a good idea `soil/carbon.md` in particular is likely to get fairly large, so splitting the content makes sense